server/models/messages.ts

```diff
@@ -0,0 +1,40 @@
+import { DataTypes, Model } from 'sequelize';
+import { sequelize } from '../database/config/connection';
+
+class MessageDatatypes extends Model {
+  declare id: number;
+  declare player_id: number;
+  declare match_id: number;
+  declare message: string;
+}
+
+export const Message = MessageDatatypes.init(
+  {
+    username: { type: DataTypes.STRING(50), allowNull: false },
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+      allowNull: false,
+    },
+    player_id: {
```

Comment:
player_id should have references to be Foreign Key

like :  player_id: {
      type: DataTypes.INTEGER,
      allowNull: false,
      references: {
        model: 'Player',
        key: 'id',
      },